src/libs/EmojiUtils.ts

```diff
@@ -322,6 +322,9 @@ function getAddedEmojis(currentEmojis: Emoji[], formerEmojis: Emoji[]): Emoji[]
  * If we're on mobile, we also add a space after the emoji granted there's no text after it.
  */
 function replaceEmojis(text: string, preferredSkinTone = CONST.EMOJI_DEFAULT_SKIN_TONE, lang: 'en' | 'es' = CONST.LOCALES.DEFAULT): ReplacedEmoji {
+    // emojisTrie importing the emoji JSON file on the app starting and we want to avoid it
```

Comment:
So instead of importing the whole file we wait until `replaceEmojis` is called?

Reply:
how much does this reduce startup time? Won't this affect user opening emoji picker as soon as app is loaded?

Reply:
yeah, we've shifted the timing of importing this file from when the thread is busy to a time when it can run more smoothly.